app/templates/base.html

```diff
@@ -1,45 +1,23 @@
-{% extends 'bootstrap/base.html' %}
-
-{% block title %}
-    {% if title %}
-        {{ title }} - Potato Mail
-    {% else %}
-        Potato Mail
-    {% endif %}
-{% endblock %}
-
-
-{% block navbar %}
-
-
-    <nav class="navbar navbar-default">
-        <div class="container-fluid">
-            <div class="navbar-header">
-                <a class="navbar-brand" >Potato Mail </a>
-            </div>
-            <div class="nav navbar-nav">
-                <li><a href="{{url_for('test') }}">Home</a></li>
-                <li><a href="{{url_for('login') }}">Login</a></li>
-                <li><a href="{{url_for('index') }}">Inbox</a></li>
-                
-                
-            </div>
-        </div>
-    </nav>
-{% endblock %}
```

Comment:
This is the nav bar that should stay.

Reply:
This makes a lot of sense to me. I've already seen this criticism so I understand that these changes are very significant to user experience.